Changelog.md

```diff
@@ -6,7 +6,6 @@ Important Bugfixes:
 
 Language Features:
  * Add `E.selector` for a non-anonymous event `E` to access the 32-byte selector topic.
- * Errors and Events allow qualified access from other contracts.
```

Comment:
This was partially wrong. #12921 only allowed access for events. Foreign errors were already accessible before.